packages/cli/src/commands/generate/graphql-client-code/generate_graphql_client_code_command.test.ts

```diff
@@ -16,11 +15,16 @@ import {
 } from '@aws-amplify/model-generator';
 import { SandboxBackendIdResolver } from '../../sandbox/sandbox_id_resolver.js';
 import { BackendIdentifierResolverWithFallback } from '../../../backend-identifier/backend_identifier_with_sandbox_fallback.js';
+import { S3Client } from '@aws-sdk/client-s3';
+import { AmplifyClient } from '@aws-sdk/client-amplify';
+import { CloudFormationClient } from '@aws-sdk/client-cloudformation';
 
 void describe('generate graphql-client-code command', () => {
-  const generateApiCodeAdapter = new GenerateApiCodeAdapter(
-    fromNodeProviderChain()
-  );
+  const generateApiCodeAdapter = new GenerateApiCodeAdapter({
```

Comment:
regarding [this comment](https://github.com/aws-amplify/amplify-backend/pull/1288/files#r1569641545), it's fine to initialize the object like this in test files